app/dashboard/views.py

```diff
@@ -1050,6 +1050,8 @@ def invoice(request):
     params['tips'] = [
         tip for tip in bounty.tips.send_happy_path() if ((tip.username == request.user.username and tip.username) or (tip.from_username == request.user.username and tip.from_username) or request.user.is_staff)
     ]
+    if (bounty.web3_created.replace(tzinfo=None) > date_to_start_charging_ten_percent.replace(tzinfo=None)):
+        params['percentage_fee'] = '10%'
```

Comment:
Ah so the percentage_fee changes per bounty as folks might get a discounted fee.
we've got a field in bounty model which stores `fee_amount` (but not %)

@danlipert what do we do here ?

Reply:
yep we have discounts associated to profiles and now cupons, so this should be dynamic

Reply:
Yes we need to calculate what the percentage fee was for a given bounty, its not always 10%. I think we should use the `fee_amount` to figure it out